tests/unit/misc/test_guiprocess.py

```diff
@@ -64,6 +64,8 @@ def test_start(proc, qtbot, message_mock, py_proc):
     assert not message_mock.messages
     assert qutescheme.spawn_output == expected
     assert proc.exit_status() == QProcess.NormalExit
+    assert proc.final_stdout().strip() == "test", proc.final_stdout()
+    assert proc.final_stderr().strip() == "", proc.final_stderr()
```

Comment:
Maybe those would fit better in `test_start_output_message` which also tests various combinations of stdout/stderr being set or unset?

Reply:
Moved to `test_start_output_message`